### PR TITLE
Created desktop file for running "add extensions" directly

### PR DESCRIPTION
**Pull request by mchf**
This implementation is done using desktop file only, however it doesn't work in ncurses

### CODE REVIEW

File: src/desktop/add_extensions.desktop [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+X-SuSE-YaST-Call=scc
+X-SuSE-YaST-Argument=select_extensions
+X-SuSE-YaST-RootOnly=yes
+X-SuSE-YaST-Group=Software
+
+Icon=yast-product-registration
+Exec=/sbin/yast2 scc select_extensions
+
+Name=Add System Extensions/Modules
+GenericName=Add System Extensions/Modules
```
Comment (lslezak) [inline src/desktop/add_extensions.desktop]: I'm not sure if using the slash is OK in the label. IIRC the old styleguide was against this and suggested using `and` or `or` words instead.

I have checked the other items and `and` is used several times, no item with slash. So I'd change it to be more consistent. (Fix all places, it's repeated several times in the file.)

File: src/desktop/add_extensions.desktop [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;X-SuSE-YaST-Software;
+
+X-SuSE-YaST-Call=scc
+X-SuSE-YaST-Argument=select_extensions
+X-SuSE-YaST-RootOnly=yes
+X-SuSE-YaST-Group=Software
+
+Icon=yast-product-registration
```
Comment (lslezak) [inline src/desktop/add_extensions.desktop]: I'd use another icon here, this is primarily for adding new extensions, the registration dialog is displayed only when the system is not registered yet.

I'd prefer the same icon as for the add-on module, i.e. `Icon=yast-addon`.

File: src/desktop/add_extensions.desktop (hunk 1 above)
Comment (mchf) [inline src/desktop/add_extensions.desktop]: just note: it worked fine in gnome even ncurses. I'll change it.

File: src/desktop/add_extensions.desktop (hunk 2 above)
Comment (mchf) [inline src/desktop/add_extensions.desktop]: no problem, I thought that someone from UX will check that before release.

File: package/yast2-registration.changes [hunk 3]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon Jan 30 20:05:56 UTC 2017 - mfilka@suse.com
+
+- fate#321044
+  - provide a shortcut for running "Add System Extensions/Modules"
```
Comment (lslezak) [inline package/yast2-registration.changes]: `or`...

Comment (mchf) [inline package/yast2-registration.changes]: here it refers to the dialog title ....

**Comment by mchf** [comment]
thanks for review

**Pull request merged**